src/generators/dart.ts

```diff
@@ -204,35 +214,45 @@ export function _toDart(requests: Request[], warnings: Warnings = []): string {
           "    ..fields[" + name + "] = " + repr(m.content, imports) + "\n";
       }
     }
+    multipart += "    ;\n\n";
```

Comment:
Thank you for catching this, but

```dart
  final req = http.MultipartRequest('POST', url)
    ..fields['username'] = 'davidwalsh'
    ..fields['password'] = 'something'
    ;
```

can't be idiomatic Dart code. Can you please change everything in the code block above to add the the newline at the beginning of the line, i.e.

```dart
  if (rawRequestObj) {
    let multipart = "http.";
    if (request.multipartUploads) {
      multipart += "MultipartRequest";
    } else {
      multipart += "Request";
    }
    multipart += "(" + repr(request.urls[0].method, imports) + ", url)";

    for (const m of request.multipartUploads || []) {
      // MultipartRequest syntax looks like this:
      //   ..fields['user'] = 'nweiz@google.com'
      // or
      // ..files.add(await http.MultipartFile.fromPath(
      //   'package', 'build/package.tar.gz',
      //   contentType: MediaType('application', 'x-tar')))
      const name = repr(m.name, imports); // TODO: what if name is empty string?
      const sentFilename = "filename" in m && m.filename;
      if ("contentFile" in m) {
        multipart += "\n    ..files.add(await http.MultipartFile.";
        if (eq(m.contentFile, "-")) {
          if (request.stdinFile) {
            multipart += "fromPath(\n";
            multipart +=
              "      " + name + ", " + repr(request.stdinFile, imports);
            if (sentFilename && request.stdinFile !== sentFilename) {
              multipart += ",\n";
              multipart += "      filename: " + repr(sentFilename, imports);
            }
            multipart += "))";
          } else if (request.stdin) {
            multipart += "fromString(\n";
            multipart += "      " + name + ", " + repr(request.stdin, imports);
            if (sentFilename) {
              multipart += ",\n";
              multipart += "      filename: " + repr(sentFilename, imports);
            }
            multipart += "))";
          } else {
            multipart += "fromString(\n";
            // TODO: read the entire thing, not one line.
            multipart +=
              "      " + name + ", stdin.readLineSync(encoding: utf8)";
            if (sentFilename) {
              multipart += ",\n";
              multipart += "      filename: " + repr(sentFilename, imports);
            }
            multipart += "))";
            imports.add("dart:io");
            imports.add("dart:convert");
          }
        } else {
          multipart += "fromPath(\n";
          multipart += "      " + name + ", " + repr(m.contentFile, imports);
          if (sentFilename && m.contentFile !== sentFilename) {
            multipart += ",\n";
            multipart += "      filename: " + repr(sentFilename, imports);
          }
          multipart += "))";
        }
      } else {
        multipart +=
          "\n    ..fields[" + name + "] = " + repr(m.content, imports);
      }
    }
    multipart += ";\n\n";
```